ChemoTracker/SignUp.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { StyleSheet, Text, TextInput, Button, View, Alert, Image } from 'react-native';
+
+export default class SignUp extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Image
+          source={require('./chemotracker.png')}
+          style={styles.logo}
+        />
+        <TextInput
+          style={styles.textFieldContainer}
+          placeholder="email"
+        />
+       <TextInput
+         style={styles.textFieldContainer}
+         placeholder="password"
+       />
+       <TextInput
+         style={styles.textFieldContainer}
+         placeholder="confirm password"
+       />
+       <View style={styles.buttonContainer}>
+         <Button
+          onPress={()=> {}}
+          title="Register"
+          color="#fff"
+          accessibilityLabel="Tap on Me"
+         />
+       </View>
+       <Text>Already Registered?  <Text style={styles.text}>LOGIN</Text></Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  textFieldContainer: {
+    height: 60,
+    width: 300,
+    backgroundColor: "#FF6666",
```

Comment:
the background is too dark, I think it should be a lighter color, so the text is more readable.. Maybe you can try opacity? if not then just use lighter color

Reply:
I agree. The red is very harsh.